cdisc_rules_engine/services/define_xml/base_define_xml_reader.py

```diff
@@ -108,10 +111,27 @@ def extract_variables_metadata(self, domain_name: str = None) -> List[dict]:
                     self._get_item_def_representation(itemdef[0], itemref, codelist_map)
                 )
         logger.info(f"Extracted variables metadata = {variables_metadata}")
+
         return variables_metadata
 
+    @cached("define-metadata-version")
+    def get_metadata_version(self, **kwargs):
+        validate_xml = kwargs.get("validate_xml", "Y")
```

Comment:
It also seems like we are using kwargs in a lot of places just to pass around the validate_xml flag. Can that not be set at object initialization so we don't have to have kwargs everywhere?